File: app/src/main/java/ru/profapp/RanobeReader/Common/ErrorConnectionException.java
package ru.profapp.RanobeReader.Common;

public class ErrorConnectionException extends Exception {

    public ErrorConnectionException(String message) {
        super(message);
    }

    public ErrorConnectionException(String message, Throwable throwable) {
        super(message, throwable);
    }

    public ErrorConnectionException() {
        super();
    }

    public ErrorConnectionException(Throwable cause) {
        super(cause);
    }
}

File: app/src/main/java/ru/profapp/RanobeReader/Common/ErrorConnectionException.kt
package ru.profapp.RanobeReader.Common

class ErrorConnectionException : Exception {

    constructor(message: String) : super(message)

    constructor(message: String, throwable: Throwable) : super(message, throwable)

    constructor() : super()

    constructor(cause: Throwable) : super(cause)
}